js/data.js

```diff
@@ -0,0 +1,40 @@
+// Capture the form and form elements
+const contactForm = document.getElementById('contact_form');
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const messageTextarea = document.getElementById('msg');
+
+// Add a form submit event
+contactForm.addEventListener('submit', (event) => {
```

Comment:
Nice job saving the form data to local storage✨. However, according to the project requirements, the data should be saved to local storage whenever the input changes, not when the form is submitted. This way, if your user is filling the form and accidentally hits refresh, all the form input fields will still contain the input data. Currently, your form data is only saved to local storage when the form is submitted. Please update it to save to local storage as the user is entering input🙏. You can do this using the `input` event listener